tests/Integration/Behaviour/Features/Context/Domain/CategoryFeatureContext.php

```diff
@@ -87,6 +90,31 @@ public function __construct()
         $this->psCatImgDir = _PS_CAT_IMG_DIR_;
     }
 
+    /**
+     * @Then I should see following root categories in ":langIso" language:
```

Comment:
If the statement is used for two different sentences, which each one has optional parameters then it shouldn't be a single function These are two different statements that deserve two functions